### ISSUE TITLE
ng upgrade fails due to incompatible Deps

### ISSUE THREAD

**Issue by niemyjski** [typebytes/angular-checklist#42]
```
➜ ng update --all
Your global Angular CLI version (8.0.0) is greater than your local
version (7.3.8). The local Angular CLI version is used.

To disable this warning use "ng config -g cli.warnings.versionMismatch false".
                  Package "angular-checklist" has an incompatible peer dependency to "@angular/common" (requires "^4.0.0" (extended), would install "8.0.0").
                  Package "angular-checklist" has an incompatible peer dependency to "@angular/core" (requires "^4.0.0" (extended), would install "8.0.0").
                  Package "angular-checklist" has an incompatible peer dependency to "rxjs" (requires "^5.1.0", would install "6.5.2").
                  Package "angular-checklist" has an incompatible peer dependency to "zone.js" (requires "^0.8.4", would install "0.9.1").
Incompatible peer dependencies found. See above.
```

**Comment by d3lm** [comment]
Thank you for opening this issue. I'll have a look when I have the time. Maybe you also want to look into this and see how you can fix this issue. We can probably fix this by passing `--force`, but we should use this with caution.

**Comment by hs2504785** [comment]
Just want to know if this project been upgraded to latest angular 11

**Comment by d3lm** [comment]
Hey! No it hasn't been upgraded yet, but I am more than happy if you submit a PR to upgrade it to the latest Angular version! That would be really cool.

My intention is not that this projects gets abandoned, so I am happy for any contribution. It's just been super busy at StackBlitz with our WebContainer release, in case you have not seen it 🔥

**Comment by geromegrignon** [comment]
closed as resolved by latest updates.
